include/fuse_common.h

```diff
@@ -984,6 +988,12 @@ int fuse_set_fail_signal_handlers(struct fuse_session *se);
  */
 void fuse_remove_signal_handlers(struct fuse_session *se);
 
+/**
+ * Config operations.
+ * These APIs are introduced in version 312 (FUSE_MAKE_VERSION(3, 12)).
+ * Using them in earlier versions will result in errors.
+ */
+#if FUSE_USE_VERSION >= FUSE_MAKE_VERSION(3, 12)
```

Comment:
Yeah, I had thought about it and should have done it from the beginning in libfuse-3.12. So let's assume we take this change and a distribution now updates libfuse with this change. Some applications linked against libfuse have might have a wrong FUSE_USE_VERSION. So now distribution updates libfuse, but does not rebuild applications - everything works fine. Later on distribution triggers a rebuild of the application (for whatever reason, like new compiler/flags/libstdc++) - out of the sudden the build fails.
I.e. I should have done it from the beginning, but I fear that it is too late right now :(

Reply:
I know that there is an issue, but we also cannot randomly fail distributions, as explained. Can you check with the alternative PR https://github.com/libfuse/libfuse/pull/1018 ? That will print a warning (i.e. in ideal case will trigger a bug report for the distribution package) and will take fields as much as possible?

Reply:
Let's keep it open for now, until the other patch is merged. In principle I would like to hear the opinion of distribution maintainers now...